Tiltfile

```diff
@@ -85,9 +79,14 @@ k8s_yaml([
 
 # Extra stuff to make debugging and testing easier
 k8s_yaml([
+  'k8s/test/namespace.yaml',
```

Comment:
resources in `ks_yaml(...)` take global precedence over anything else defined in the `Tiltfile`, so I moved namespace creation here so that it's available for these resources
(creation of the new configmaps was erroring in CI, but not locally for some reason)